### DISCUSSION TITLE
git hub account password not recovering!

### DISCUSSION THREAD

**Discussion by samcorp024** [community/community#138354] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hey any one can help me

actually i am unable to ricover my github password 

and i  am not getting password recover link on my mail why?

**Comment by ACoastGame** [comment]
I think something might be wrong with your network. if that's the case the GitHub email will eventually get to your inbox later on just keep refreshing your inbox. if this is not the case on your side then I'd suggest contacting the git team [here](https://support.github.com/).
